src/ert/config/queue_config.py

```diff
@@ -245,88 +247,52 @@ def create_local_copy(self) -> QueueConfig:
         )
 
     @property
-    def max_running(self) -> int:
-        max_running = 0
-        for key, val in self.queue_options.get(self.queue_system, []):
-            if key == "MAX_RUNNING":
-                max_running = int(val)
-        return max_running
+    def selected_queue_options(self) -> Dict[str, str]:
```

Comment:
Is it worth creating a type alias for `QueueOption`?
`QueueOption = Literal['QUEUE_SYSTEM', 'REALIZATION_MEMORY', 'MAX_SUBMIT']`